tests/epyccel/test_epyccel_functions.py

```diff
@@ -13,6 +13,57 @@ def clean_test():
     shutil.rmtree('__pycache__', ignore_errors=True)
     shutil.rmtree('__epyccel__', ignore_errors=True)
 
+def test_func_no_args_1(language):
+    '''test function with return value but no args'''
+    def free_gift():
+        gift = 10
+        return gift
+
+    c_gift = epyccel(free_gift, language=language)
+    assert free_gift() == c_gift()
+    assert isinstance(free_gift(), type(c_gift()))
+    unexpected_arg = 0
+    with pytest.raises(TypeError):
+        c_gift(unexpected_arg)
+
+def test_func_no_args_2(language):
+    '''test function with negative return value but no args'''
+    def p_lose():
+        lose = -10
+        return lose
+
+    c_lose = epyccel(p_lose, language=language)
+    assert p_lose() == c_lose()
+    assert isinstance(p_lose(), type(c_lose()))
+    unexpected_arg = 0
+    with pytest.raises(TypeError):
+        p_lose(unexpected_arg)
```

Comment:
This should be `c_lose`

Reply:
yes, the names were a little bit similar. I think I need to change my way in naming

Reply:
Given that they essentially represent the same thing it's not really surprising that the names are so close. I think there are several cases like this in the tests folder. I wouldn't worry about it too much. This is why we have a review process for pull requests. The more people look at the code before it is merged into the master branch, the more likely we are to catch this kind of thing

Reply:
yes, so I think I need to request more reviewers like my friends from 1337, it's an occasion for them to learn more about pyccel.